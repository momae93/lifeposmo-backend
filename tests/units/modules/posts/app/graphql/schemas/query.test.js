const querySchemas = require('@PostApp/graphql/schemas/query');

describe('[MODULES][POSTS][APP][GRAPHQL - SCHEMAS] : query schemas', () => {
  it('should returns schema of queries', async () => {
    // FUNCTION TESTED
    expect(querySchemas).toMatchSnapshot();
  });
});
