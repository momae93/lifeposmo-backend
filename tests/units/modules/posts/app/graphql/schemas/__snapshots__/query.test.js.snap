// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[MODULES][POSTS][APP][GRAPHQL - SCHEMAS] : query schemas should returns schema of queries 1`] = `
Object {
  "definitions": Array [
    Object {
      "directives": Array [],
      "fields": Array [
        Object {
          "arguments": Array [],
          "description": undefined,
          "directives": Array [],
          "kind": "FieldDefinition",
          "name": Object {
            "kind": "Name",
            "value": "posts",
          },
          "type": Object {
            "kind": "ListType",
            "type": Object {
              "kind": "NonNullType",
              "type": Object {
                "kind": "NamedType",
                "name": Object {
                  "kind": "Name",
                  "value": "Post",
                },
              },
            },
          },
        },
        Object {
          "arguments": Array [
            Object {
              "defaultValue": undefined,
              "description": undefined,
              "directives": Array [],
              "kind": "InputValueDefinition",
              "name": Object {
                "kind": "Name",
                "value": "id",
              },
              "type": Object {
                "kind": "NamedType",
                "name": Object {
                  "kind": "Name",
                  "value": "Int",
                },
              },
            },
          ],
          "description": undefined,
          "directives": Array [],
          "kind": "FieldDefinition",
          "name": Object {
            "kind": "Name",
            "value": "post",
          },
          "type": Object {
            "kind": "NamedType",
            "name": Object {
              "kind": "Name",
              "value": "Post",
            },
          },
        },
        Object {
          "arguments": Array [
            Object {
              "defaultValue": undefined,
              "description": undefined,
              "directives": Array [],
              "kind": "InputValueDefinition",
              "name": Object {
                "kind": "Name",
                "value": "idAuthor",
              },
              "type": Object {
                "kind": "NamedType",
                "name": Object {
                  "kind": "Name",
                  "value": "Int",
                },
              },
            },
          ],
          "description": undefined,
          "directives": Array [],
          "kind": "FieldDefinition",
          "name": Object {
            "kind": "Name",
            "value": "postsWrittenBy",
          },
          "type": Object {
            "kind": "ListType",
            "type": Object {
              "kind": "NonNullType",
              "type": Object {
                "kind": "NamedType",
                "name": Object {
                  "kind": "Name",
                  "value": "Post",
                },
              },
            },
          },
        },
      ],
      "interfaces": Array [],
      "kind": "ObjectTypeExtension",
      "name": Object {
        "kind": "Name",
        "value": "Query",
      },
    },
  ],
  "kind": "Document",
  "loc": Object {
    "end": 113,
    "start": 0,
  },
}
`;
